src/System.Management.Automation/FormatAndOutput/out-console/ConsoleLineOutput.cs

```diff
@@ -163,6 +87,11 @@ internal sealed class ConsoleLineOutput : LineOutput
         internal static readonly PSTraceSource tracer = PSTraceSource.GetTracer("ConsoleLineOutput", "ConsoleLineOutput");
         #endregion tracer
 
+        /// <summary>
+        /// The default buffer cell calculation already works for the PowerShell console host and Visual studio code host.
+        /// </summary>
+        private static readonly HashSet<string> s_psHost = new(StringComparer.Ordinal) { "ConsoleHost", "Visual Studio Code Host" };
```

Comment:
@SeeminglyScience Now that the default buffer cell calculation takes into account escape sequences and CJK characters, I think we can make `ConsoleHost` and VSCode host to use the default `DisplayCells` instead of calling into the host specific `LengthInBufferCells` method. Let me know if you have any concerns.

Reply:
> @SeeminglyScience Now that the default buffer cell calculation takes into account escape sequences and CJK characters, I think we can make `ConsoleHost` and VSCode host to use the default `DisplayCells` instead of calling into the host specific `LengthInBufferCells` method. Let me know if you have any concerns.

I agree! VSCode just redirects to `ConsoleHost*`'s implementation so shouldn't need any changes there.

Also does this mean that piping to line output writers that aren't host aware should work now too? 😮 (e.g. `Out-String` and `less.exe` would calculate the wrong value width because they never call `LengthInBufferCells`)